### DISCUSSION TITLE
Modify river network for the Sag domain

### DISCUSSION THREAD

**Discussion by changliao1025** [DOE-ICoM/liao-etal_2023_mosart_joh#14] [Ideas]
The existing input files have flaws in the coverage, which caused missing contributing areas.
I will manually modify this input and re-run the simulation.

In the future, we need to better river networks and DEM for some sensitive domains.

**Comment by dengwirda** [comment]
@changliao1025, I'm hoping the upcoming [v2 revision to HydroSHEDS](https://www.hydrosheds.org/hydrosheds-v2) may be an option that improves global network fidelity.

**Reply by changliao1025** [reply]
Great, at some point, we should discuss also including lakes and watershed boundaries in the workflow.
